docs/guides/kubernetes/deploy-tobs-on-linode-kubernetes-engine/index.md

```diff
@@ -0,0 +1,174 @@
+* [Promscale](https://docs.timescale.com/promscale/latest/about-promscale/) provides the translation layer between Prometheus and the database. It allows the Prometheus server to store and retrieve metrics from TimescaleDB, and allows users to use PromQL on Promscale, and Prometheus.
+* [Prometheus](https://prometheus.io/docs/introduction/overview/) is an open-source systems monitoring and altering stack. It has become the de-facto standard in metric monitoring and is the basis of standards such as OpenMetrics. It allows you to monitor and understand how your infrastructure and applications are performing. Service discovery allows Prometheus to automagically discover components within your Kubernetes cluster that are already emitting metrics.
+* [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics) exports the metrics related to Kubernetes resources such as the status and count of Kubernetes resources, with visibility of the desired resources and the current resources, as well as the trends in your cluster.
+* [Node-Exporter](https://github.com/prometheus/node_exporter) is deployed to export node related metrics such as cpu, memory, and others from the Kubernetes cluster.
+
+## Before You Begin
+
+{{< note >}}
+This guide was written using [Kubernetes version 1.23](https://v1-17.docs.kubernetes.io/docs/setup/release/notes/).
+{{< /note >}}
+
+1.  [Deploy an LKE Cluster](/docs/guides/deploy-and-manage-a-cluster-with-linode-kubernetes-engine-a-tutorial/). This guide was written using an example node pool with three [2 GB Linodes](https://www.linode.com/pricing/). Depending on the workloads you plan to deploy on your cluster, you may consider using Linodes with more available resources.
+
+1.  Install [Helm 3](/docs/kubernetes/how-to-install-apps-on-kubernetes-with-helm-3/#install-helm) to your local environment.
+
+1.  Install [kubectl](/docs/guides/deploy-and-manage-a-cluster-with-linode-kubernetes-engine-a-tutorial/#install-kubectl) to your local environment and [connect to your cluster](/docs/guides/deploy-and-manage-a-cluster-with-linode-kubernetes-engine-a-tutorial/#connect-to-your-lke-cluster-with-kubectl).
+
+1.  Create the `monitoring` namespace on your LKE cluster:
+
+        kubectl create namespace monitoring
+
+1.  Add the stable Helm charts repository to your Helm repos:
+
+        helm repo add stable https://charts.helm.sh/stable
+
+1.  Update your Helm repositories:
+
+        helm repo update
+
+
+## TOBS Minimal Deployment
+
+In this section, learn to deploy TOBS for individual/local access with `kubectl` [Port-Forward](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#port-forward).
+
+### Deploy The Observability Stack
+1. Install a certificate manager for your LKE cluster:
```

Comment:
Need an extra return after "Install a certificate manager for your LKE cluster:" for the following command to render as a command block